src/app/starredentries/starredentries.component.html

```diff
@@ -144,7 +139,7 @@ <h4>
                 <span>
                   <mat-chip-list>
                     <mat-chip class="bubble workflow-background" *ngFor="let label of workflow?.labels | slice: 0:3">{{
-                      label.value
+                      $any(label).value
```

Comment:
This code is inside an `ng-template`, and when you do that, you lose type information. An ugly `any`, alas. Only way to workaround it is to have a component instead of a template, I believe.